OloEditor/src/Panels/PostProcessSettingsPanel.cpp

```
@@ -31,6 +40,30 @@ namespace OloEngine
         DrawDOFSection();
         DrawMotionBlurSection();
 
+        // Undo tracking: detect changes and push command when editing ends
+        if (m_CommandHistory)
+        {
+            const bool changed = (std::memcmp(&m_Snapshot, &settings, sizeof(PostProcessSettings)) != 0);
```

Clang (14.0.6)

[warning] variable 'changed' is not initialized

(cppcoreguidelines-init-variables)